app/styles/variables.less

```diff
@@ -1,4 +1,4 @@
 // copied from origin-web-console
 @panel-shaded: #f2f2f2;
 
-@navbar-height: 69px;
+@navbar-height: 72px;
```

Comment:
I'd rather use flex box so that there is no need to keep track of the navbar height.